styx-scheduler-service/src/test/java/com/spotify/styx/flyte/FlytePhaseTest.java

```diff
@@ -45,7 +45,7 @@ public void testSuccessfulMapping() {
 
   @Test
   public void testUnknownMapping() {
-    assertThrows(NoSuchElementException.class, () -> FlytePhase.fromProto(Execution.WorkflowExecution.Phase.UNDEFINED));
+    assertThrows(NullPointerException.class, () -> FlytePhase.fromProto(Execution.WorkflowExecution.Phase.forNumber(100)));
```

Comment:
What does 100 mean?

Reply:
i thought it would work but it won't. it supposed to pick up the n-th item from the enum, but it just throws a NPE